tiup/tiup-cluster-topology-reference.md

```diff
@@ -463,6 +464,50 @@ tiproxy_servers:
   - host: 10.0.1.22
 ```
 
+### `kvcdc_servers`
+
+`kvcdc_servers` specifies the machines to which the [TiKV-CDC](https://tikv.org/docs/7.1/concepts/explore-tikv-features/cdc/cdc/) services are deployed. It also specifies the service configuration on each machine. `kvcdc_servers` is an array. Each array element contains the following fields:
+
+- `addr`: The listening address of TiKV-CDC, used to provide HTTP API and Prometheus queries. The default value is `127.0.0.1:8600`.
+
+- `advertise-addr`: The advertised address via which clients access TiKV-CDC. If not specified, the value is the same as that of `addr`.
+
+- `pd`: A comma-separated list of PD endpoints.
+
+- `config`: The address of the configuration file that TiKV-CDC uses (optional).
+
+- `data-dir`: The directory that TiKV-CDC uses to store temporary files for sorting. It is recommended to ensure that the free disk space for this directory is greater than or equal to 500 GiB (optional).
+
+- `gc-ttl`: The TTL (Time to Live, in seconds) of the service-level GC safepoint in PD set by TiKV-CDC (optional). It is the duration that replication tasks can be suspended, defaulting to `86400`, which is 24 hours. Note that suspending replication tasks affects the progress of TiKV garbage collection. The longer the `gc-ttl`, the longer changefeeds can be paused, but at the same time, more obsolete data will be kept and occupy more space. Vice versa.
+
+- `log-file`: The path to which logs are output when the TiKV-CDC process is running (optional). If not specified, logs are written to the standard output (stdout).
+
+- `log-level`: The log level when the TiKV-CDC process is running (optional). The default value is `"info"`.
+
+- `ca`: The path of the CA certificate file in PEM format for TLS connection (optional).
+
+- `cert`: The path of the certificate file in PEM format for TLS connection (optional).
+
+- `key`: The path of the private key file in PEM format for TLS connection (optional).
+
+- `cert-allowed-cn`: The allowed caller identities (certificate Common Name, optional). Use commas to separate multiple CNs.
+
+For the above fields, you cannot modify these configured fields after the deployment:
```

Comment:
Sync the following with the Chinese version if updated.

Reply:
done in [`3714c46` (#19230)](https://github.com/pingcap/docs/pull/19230/commits/3714c4678533c4654c7fdd77c47dfa4d654b8c7e)